cmd/dimension.go

```diff
@@ -60,7 +60,7 @@ var listDimensionsCmd = &cobra.Command{
 	Run: func(ccmd *cobra.Command, args []string) {
 		state := internal.PrepareEngineState(rootCtx, headers, false)
 		items := internal.ListDimensions(state.Ctx, state.Doc)
-		printer.PrintNamedItemsList(items, viper.GetBool("bash"))
+		printer.PrintNamedItemsList(items, viper.GetBool("bash"), false)
```

Comment:
Do we only want to print the title for variables?

Reply:
With the bash-flag yes, since we can only retrieve variables by name. Normally the print for bash completion only prints IDs so it had to be modified to support this case.